### PR TITLE
cms: import and export spec

### PR DESCRIPTION
**Pull request by Steven-Chan**
connects #151

### CODE REVIEW

File: features/151-cms-import-export/api.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+  - if the `_id` value is not found
+    - a new record will be created
+
+#### Import behaviour
+
+The import API would iterate through items in the file and save them to server. The operation is non-atomic, i.e. for one API call, some records may be saved successfully while some may not.
+
+If there are multiple items that share the same identifier, only one of them are effectively saved to the server.
+
+#### Custom identifer for Record
```
Comment (rickmak) [inline features/151-cms-import-export/api.md]: s/identifer/identifier/g

File: features/151-cms-import-export/api.md [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+|---|---|
+|1|1|
+|1|2|
+
+Config:
+```
+records:
+  A:
+    (skip)
+    import:
```
Comment (carmenlau) [inline features/151-cms-import-export/api.md]: I think this should be export?

File: features/151-cms-import-export/api.md [hunk 3]
```diff
@@ -0,0 +1,562 @@
...
+      <td>Number</td>
+      <td>Number</td>
+    </tr>
+    <tr>
+      <td>Boolean</td>
+      <td>
+        <ul>
+          <li>True/False (Default)</li>
+          <li>Yes/No</li>
+          <li>1/0</li>
```
Comment (carmenlau) [inline features/151-cms-import-export/api.md]: For all of these options, will we give them a name? Or what value would we expect user will fill-in in yaml?

**Comment by Steven-Chan** [comment]
@carmenlau updated

**Pull request merged**